### PR TITLE
feat(@core/quote) : add quote

### PR DESCRIPTION
**Pull request by marobakr**
1. add a quote section

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -2,3 +2,8 @@
 @import "variables.scss";
 @import "base.scss", "typography.scss";
 @import "mixins.scss";
+
+/*the background i put it just to show the final result  */
+body {
```
Comment (vcpc) [inline src/styles.scss:7]: This should be in te `base/` folder at `_backgrounds.scss` file
And remove that `_base.scss`

**Comment by marobakr** [comment]
1. add quote
1.1.remove bg color

File: src/app/core/components/quote/quote.component.scss [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    /* Box Model */
+    width: 41px;
+    height: 28px;
+    padding: 4px 8px;
+
+    transform: translateY(-50%);
+    top: 0;
+
+    /* Color */
+    background-color: #282c33;
```
Comment (vcpc) [inline src/app/core/components/quote/quote.component.scss:113]: use the `$background` variable

File: src/app/core/components/quote/quote.component.scss [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    * 1. Change Positions
+    * 2. Change Box Model
+    * 3. Change Text Size
+*/
+  @media screen and (max-width: 567px) {
+    /* Postion*/
+    right: 0px; /* 1 */
+
+    /* Box Model*/
+    padding: {
```
Comment (vcpc) [inline src/app/core/components/quote/quote.component.scss:27]: lower the padding more

File: src/app/core/components/quote/quote.component.scss [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+      /* 1 */
+      left: 20px;
+      right: 20px;
+    }
+
+    p {
+      /* Text */
+      font-size: 12px; /* 1 */
```
Comment (vcpc) [inline src/app/core/components/quote/quote.component.scss:35]: this should be 3 not one

**Pull request closed without merging**